.drone.yml

```diff
@@ -7,16 +7,29 @@ platform:
 
 steps:
 - name: cache:restore
-  image: node:12
+  image: codercom/nbin:centos
```

Comment:
We're running a cacher script here I believe which we shouldn't need the extra SDK for.

Reply:
Ah yeah so the script runs yarn after restoring the node modules which means it needs to be the same as the build step for the native modules. 

The "proper" solution is probably to set YARN_CACHE_FOLDER in the Drone config, then the cacher script caches that, then add a separate yarn step in the Drone config, and then also set YARN_CACHE_FOLDER when running the build script.

It's a lot of duplication among the six (now four) builds so I didn't feel like doing it and just put it in the script for now.

Reply:
And to clarify why I'm not just running yarn in the build script anymore, it's because I wanted to run the linter and the tests before doing the build.